jinja/carnival/trivia_party.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% block content %}
+{% block navbar %}
+{% include "navbar.html" %}
+{% endblock %}
+
+<div class="parallax" style="background-image: url('../img/xanadu-background.png'); box-shadow: none;">
+    <div class="container">
+
+        <div class="mt-5 pt-5 text-center teal-text text-lighten-2">
+            <img src="../img/carnival_logo_splash.png" class="logo"></img>
+        </div>
+        <div class="row text-center">
+            <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">
+            A celebration needs a party so come to the Quantum Carnival's very own <strong>Trivia Party!</strong>
+            </h3>
+	<div style="font-size: 14pt;">
+            <p>
+            Are you ready to put your knowledge to the test?
+</p>
+
+<p>
+Come to the Trivia Party on November 12th at noon ET and have fun while learning new things!
```

Comment:
at noon (12pm ET)
or link to time converter:
https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250